dev-docs/feature-conditional.md

```diff
@@ -0,0 +1,239 @@
+---
+layout: page
+title: Conditional Ad Units - <font color="red">NEW!</font>
+description: Using labels for conditional bids
+---
+
+<div class="bs-docs-section" markdown="1">
+
+# Conditional Bids and AdUnits
+
+Prebid 1.0 introduced the `sizeConfig` feature primarily for [responsive ad designs]({{site.baseurl}}/dev-docs/feature-responsive.html), focusing on constraining sizes based on device.
+But a number of other scenarios are supported as well:
+
+1. What if some bidders should be skipped for some devices?
+1. What if some bidders have different parameters for different devices?
+1. What if some AdUnit auctions should be skipped entirely for some devices?
+1. What if some bid requests apply only to users originating from certain countries? 
+
+By supporting these scenarios, header bidding can be more efficient - the browser can send bids to a more surgical set of bidders based on device size or other attributes the page code can create.
+
+Before going into how these scenarios work, a few words about the design of Prebid's approach. With Prebid 1.0,
```

Comment:
Same comment as above re: referring explicitly to Prebid 1.0 (text like this is timely now but will have to be removed later when 1.0 is no longer new, and we are likely to forget to remove it).